### ISSUE TITLE
Allow scope() to be used for operations

### ISSUE THREAD

**Issue by bparise** [feathersjs/feathers-sequelize#130]
I want to scope() records through hooks.  I use scopes frequently in my API to allow users with certain roles access to mutate objects.  I know I can just where={}, but scopes are way more convenient and centralized to the model.

Would be nice to:

```js
hook => {
  hook.params.sequelize.scope = 'approved'
  return Promise.resolve(hook)
}
```

Right now, this setting is ignored because you have to scope records like:
```js
Model.scope('approved').findAll()...
```

I can do a PR for this, just let me know!
```js
let Model = this.Model
if (params.scope) {
  Model = this.Model.scope(params.scope)
}
```

**Comment by DesignByOnyx** [comment]
@bparise - If you want to submit a PR. I will endorse it. This would be a great feature! If you don't have time I will try and squeeze it in - let me know. If you want to do it, make sure to write tests and update the docs. You go to a particular page at `https://docs.feathersjs.com/`, then click the "Edit this page" link in the top.

**Comment by bparise** [comment]
Done!  I'll update the documentation once the PR is officially accepted.

**Comment by daffl** [comment]
Closed via #142
